ArduCopter/heli.cpp

```diff
@@ -200,4 +213,11 @@ void Copter::heli_radio_passthrough()
     motors.set_radio_passthrough(channel_roll->control_in, channel_pitch->control_in, channel_throttle->control_in, channel_yaw->control_in);
 }
 
+void Copter::set_using_governor_switch(bool b)
```

Comment:
This could be skipped if you agree with https://github.com/diydrones/ardupilot/pull/3343/files#r47831314